src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,171 @@
+
+## Verify a license
+
+To verify a license, open the [built-in SQL shell]({% link {{ page.version.version }}/cockroach-sql.md %}) and use the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster-setting.md %}) command to check the license key:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING enterprise.license;
+~~~
+~~~
+             enterprise.license
++-------------------------------------------+
+  crl-0-ChB1x...
+(1 row)
+~~~
+
+The license setting is also logged in the cockroach.log on the node where the command is run:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+$ cat cockroach.log | grep license
+~~~
+~~~
+I171116 18:11:48.279604 1514 sql/event_log.go:102  [client=[::1]:56357,user=root,n1] Event: "set_cluster_setting", target: 0, info: {SettingName:enterprise.license Value:xxxxxxxxxxxx User:root}
+~~~
+
+## Monitor for license expiry
+
+You can monitor the time until your license expires in the following ways:
+
+1. [Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}): The `seconds_until_enterprise_license_expiry` metric reports the number of seconds until the license on a cluster expires. It will report `0` if there is no license, and a negative number if the license has already expired. For more information, see [Monitoring and Alerting]({% link {{ page.version.version }}/monitoring-and-alerting.md %}).
+1. [DB console]({% link {{ page.version.version }}/ui-overview.md %}): Several [license expiration messages]({% link {{ page.version.version }}/ui-overview.md %}#license-expiration-message) may be displayed, depending on the status of your cluster's license.
+1. [CockroachDB {{ site.data.products.cloud }} Console][cloud-console]: If you have an **Enterprise Free** or **Enterprise Trial** cluster, you will see notifications in the **Enterprise Licenses** section, as well as receive notification emails sent to users with **Org Admin** permissions.
+1. CockroachDB emits [log messages]({% link {{ page.version.version }}/logging-overview.md %}) when a cluster is at risk of being [throttled](#throttling) due to license expiration or telemetry requirements. The database will also return notices to [SQL clients]({% link {{ page.version.version }}/cockroach-sql.md %}) or [applications]({% link {{ page.version.version }}/install-client-drivers.md %}) that try to execute [transactions]({% link {{ page.version.version }}/transactions.md %}).
+
+{{site.data.alerts.callout_info}}
+During the transition to the [CockroachDB Software License][csl], expiration behavior (including [throttling](#throttling)) will work as follows depending on your version of CockroachDB.
+
+For versions v23.1.29, v23.2.15, v24.1.7, v24.2.5, and v24.3.0 and later, clusters will behave as described in this documentation.
+
+For versions less than or equal to v23.1.28, v23.2.14, v24.1.6, and v24.2.4, the behavior will be as follows:
```

Comment:
this one i'm feeling a little squirrely about due to the legal review already having happened, will leave it there for now 

for example i'd hate for "and earlier" to be construed as only referring to v24.2.4 instead of what i know you obv. mean which is "x.xx and earlier, y.yy and earlier, z.zz and earlier", etc.

but IANAL so i'm gonna leave this one